src/test/java/com/epam/jamp2/service/CalculationCommandExecutionServiceTest.java

```diff
@@ -48,7 +48,12 @@ public void should_pass_4_different_ccy()
         CalculationCommand cmd = mockCalculationCommand(ADD_DIFF_CCY, 686.6, 5.0, FxRateMockData.CNY);
 
         Value result = calculationCommandExecutionServiceImpl.calculate(cmd);
-        validateCalculatedValue(result, FxRateMockData.CNY, 691.6, 0.1);
+
+        Assert.assertNotNull(result);
+        Assert.assertThat(result.getCurrencyCode().get(), is(FxRateMockData.CNY));
+        Assert.assertThat(result.getValue(),
+                is(closeTo(new BigDecimal(691.6), new BigDecimal(0.1))));
```

Comment:
When you know exact expected value the usage of closeTo() matcher is discouraged, better to check for strict equality